app/views/directives/deploy-image.html

```diff
@@ -73,7 +73,8 @@
         administrator.
       </div>
       <h2>
-        {{app.name}}<span ng-if="import.tag">:{{import.tag}}</span>
+        <span ng-if="mode === 'dockerImage'">{{app.name}}<span ng-if="import.tag">:{{import.tag}}</span></span>
```

Comment:
Yeah we can change it for the other case. The difference is that we will actually create an image stream tag with that value in your project for `mode === 'dockerImage'`